awx/main/models/unified_jobs.py

```diff
@@ -630,6 +630,13 @@ class Meta:
         help_text=_("The date and time the job finished execution."),
         db_index=True,
     )
+    canceled_on = models.DateTimeField(
+        null=True,
+        default=None,
+        editable=False,
+        help_text=_("The date and time the job was canceled."),
```

Comment:
might be better to say "the date and time of the request ...", but different people have different ideas about how general vs. accurate the OPTIONS help should be.